services/device-registry-file/src/test/java/org/eclipse/hono/deviceregistry/file/FileBasedRegistrationServiceTest.java

```diff
@@ -87,6 +90,21 @@ public void setUp() {
 
         registrationService = new FileBasedRegistrationService(vertx);
         registrationService.setConfig(registrationConfig);
+
+        final AutoProvisioner autoProvisioner = new AutoProvisioner(registrationService, registrationService);
+        autoProvisioner.setConfig(new AutoProvisionerConfigProperties());
+
+        final DownstreamSenderFactory downstreamSenderFactoryMock = mock(DownstreamSenderFactory.class);
+        when(downstreamSenderFactoryMock.connect()).thenReturn(Future.succeededFuture());
+        doAnswer(invocation -> {
+            final Handler<AsyncResult<Void>> handler = invocation.getArgument(0);
+            handler.handle(Future.succeededFuture());
+            return null;
+        }).when(downstreamSenderFactoryMock).disconnect(any(Handler.class));
```

Comment:
`VertxMockSupport.anyHandler()` can be used instead of `any(Handler.class)` to prevent the "unchecked assignment" warning.

Reply:
Thanks for the hint 👍